librz/bin/p/bin_pe.inc

```diff
@@ -610,6 +610,7 @@ static RzBinInfo *info(RzBinFile *bf) {
 	ret->machine = PE_(rz_bin_pe_get_machine)(bf->o->bin_obj);
 	ret->subsystem = PE_(rz_bin_pe_get_subsystem)(bf->o->bin_obj);
 	ret->default_cc = PE_(rz_bin_pe_get_cc)(bf->o->bin_obj);
+	ret->linker = PE_(rz_bin_pe_get_linker)(bf->o->bin_obj);
```

Comment:
Ok, i gave a thought and probably we should just use the `compiler` field.

change that `rz_str_newf("%02d.%02d", major, minor);` into `rz_str_newf("Linker %02d.%02d", major, minor);`

Reply:
Ok just to be clear, we have to print it in the table as `compiler  Linker XX.XX`
And already existing field `compiler` will hold `Linker XX.XX`

Reply:
yes. i don't see compiler being set on pe, so you can fill it with the linker data.